cypress/integration/create-user.spec.js

```diff
@@ -58,91 +56,6 @@ describe("Creation of new user", () => {
-    const emailAddress = "bichardemail1@example.com"
-    const newPassword = "bichardemail1"
-    cy.task("getPasswordResetCode", emailAddress).then((passwordResetCode) => {
-      const newPasswordToken = generateNewPasswordToken(emailAddress, passwordResetCode)
-      cy.visit(`/login/new-password?token=${newPasswordToken}`)
-      cy.get("input[type=password][name=newPassword]").type(newPassword)
-      cy.get("input[type=password][name=confirmPassword]").type(newPassword)
-      cy.get("button[type=submit]").click()
-      cy.get('[data-test="error-summary"]').contains("Password contains personal information.")
-    })
-  })
-
-  it("should allow the new user to set their password via a link", () => {
-    const emailAddress = "bichardemail1@example.com"
-    const newPassword = "Test@123456"
-    cy.task("getPasswordResetCode", emailAddress).then((passwordResetCode) => {
-      const newPasswordToken = generateNewPasswordToken(emailAddress, passwordResetCode)
-      cy.visit(`/login/new-password?token=${newPasswordToken}`)
-      cy.get("input[type=password][name=newPassword]").type(newPassword)
-      cy.get("input[type=password][name=confirmPassword]").type(newPassword)
-      cy.get("button[type=submit]").click()
-      cy.get('[data-test="success-banner_heading"]').should("have.text", "You can now sign in with your new password.")
-    })
-  })
-
-  it("should fail to set password and provide new user with feedback when password is too short", () => {
-    const emailAddress = "bichardemail1@example.com"
-    const newPassword = "shorty"
-    cy.task("getPasswordResetCode", emailAddress).then((passwordResetCode) => {
-      const newPasswordToken = generateNewPasswordToken(emailAddress, passwordResetCode)
-      cy.visit(`/login/new-password?token=${newPasswordToken}`)
-      cy.get("input[type=password][name=newPassword]").type(newPassword)
-      cy.get("input[type=password][name=confirmPassword]").type(newPassword)
-      cy.get("button[type=submit]").click()
-      cy.get('[data-test="error-summary"]').contains("Password is too short.")
-    })
-  })
-
-  it("should fail to set password and provide new user with feedback when password is banned", () => {
-    const emailAddress = "bichardemail1@example.com"
-    const newPassword = "password"
-    cy.task("getPasswordResetCode", emailAddress).then((passwordResetCode) => {
-      const newPasswordToken = generateNewPasswordToken(emailAddress, passwordResetCode)
-      cy.visit(`/login/new-password?token=${newPasswordToken}`)
-      cy.get("input[type=password][name=newPassword]").type(newPassword)
-      cy.get("input[type=password][name=confirmPassword]").type(newPassword)
-      cy.get("button[type=submit]").click()
-      cy.get('[data-test="error-summary"]').contains("Password is too easy to guess.")
-    })
-  })
-
-  it("should respond with forbidden response code when CSRF tokens are invalid in new password page", (done) => {
```

Comment:
I notice that there are a few missing tests in the new format/flow. Is the csrf check not valid anymore?

Reply:
This test was removed because the subject of the test was also removed (the new-password page).

If we have been testing functionality that isn't related solely to the new-password view, then I would say this is a problem because that would mean the subject of these tests is not a page but a piece of functionality.

All I have done as part of the new flow is directed the user towards the reset-password view, of which they are using that reset password view for its' intended, original purpose; there should not be any requirement to add any additional tests around this flow (all tests have already been written)?